### PR TITLE
Upgraded frequency-chain/api-augment to 0.9.29-rc1

### PR DESCRIPTION
**Pull request by dopatraman**
Problem
=======
The goal of this PR is to upgrade the `api-augment` package to the recent RC version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/LibertyDSNP/schemas#readme",
   "dependencies": {
-    "@frequency-chain/api-augment": "^0.0.0-924aaa",
+    "@frequency-chain/api-augment": "0.9.29-rc1",
```
Comment (saraswatpuneet) [inline package.json:24]: We should leave "^"

**Review by saraswatpuneet — APPROVED** [review]
One comment

**Comment by saraswatpuneet** [comment]
@dopatraman  or @rlaferla can we also check and update name of extrinsic if it has changed after naming update.


https://github.com/LibertyDSNP/schemas/blob/84c5b99050e971093402c7ca4bfb5e08b5113956/deploy.ts#L86

**Review by wilwade — CHANGES_REQUESTED** [review]
I got a `TypeError: api.tx.schemas.registerSchema is not a function` when I tried to run it. I think we changed the name to createSchema right? Although not sure why it didn't fail on the ts build...

**Comment by dopatraman** [comment]
@wilwade how did you run it? I ran both the build and the tests, both worked.

**Comment by wilwade** [comment]
> @wilwade how did you run it? I ran both the build and the tests, both worked.

I ran the deploy against instant sealing: https://github.com/LibertyDSNP/schemas#deploy

**Review by wilwade — APPROVED** [review]

**Pull request merged**